### DISCUSSION TITLE
2022 Academic Conference Workshop, 2022-06-24 속초

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#229] [Workshop]
안녕하세요.

올해도 작년과 마찬가지로 workshop이 진행됩니다.
지난 모임 때 오신 분들에게는 간단히 설명 드렸는데 워크샵 장소, 구체적인 내용과 일정, 그리고 비용 등에 대해 정리해 보려고 합니다.

## 내용

2022 Academic Conference workshop은 크게 아래 3 개의 주제로 이야기 할 예정입니다.

- 올해 읽었던 책 세 권에 대한 내용 중 기억 나는 것에 대한 이야기
- 책을 읽고 난 후 나의 변화
- 2023 Academic Conference에 선정된 책 preview 및 최종 결정

## 장소

- 롯데리조트 속초
  - 좋은 숙소를 예약하고 초대 하려고 했는데, 이미 스위트룸은 다 예약이 차고 없네요 ㅜㅜ
  - 저는 롯데리조트 속초 디럭스 더불룸을 예약 했습니다. 이것도 마감될까봐 빨리 예약했어요.
- 롯데리조트 속초 인근 펜션
  - 롯데리조트 속초의 가격이 부담되시는 분들은, 롯데리조트 속초 도보 10분 거리에 펜션들이 있습니다.
  - 숙소는 거기로 잡고 얘기하거나 식사할때만 같이 모여도 될 것 같습니다.

### 6월 24일 금요일 

- 오후 12시
  - 서울 용산 출발, 양양고속도로 이용 오후  3시 전후로 롯데리조트 속초 도착
  - 어떻게 모여서 출발할지, 중간 pick up은 어떻게 할지 논의 필요
  - 홍천 or 내린천 휴게소 들러서 간식 타임
- 오후 4시
  - 짐 정리 및 잠깐 휴식 후 롯데리조트 속초 R.9PUB에서 워크샵 진행
- 오후 6시
  - 속초아이 관광
  - 롯데리조트 속초에서 도보 20분 거리
  - 오는 길에 야식으로 먹을 만석닭강정 포장 예정
- 오후 8시
  - 저녁식사, 외웅치항 근처 자매네 횟집 추천 (오징어회 맛있습니다)
- 오후 10시
  - 대충 폭죽놀이 진행
  - 그런 분위기 아니면 안터뜨리는 걸로... 
- 오후 11시
  - 모여서 짧은 2차 뒷풀이 진행 with 만석 닭강정
  - 집에 모셔 놓은 글렌피딕 21년산 개봉합니다.

### 6월 25일 토요일

- 오전 8시
  - 조식 타임
- 오전 9시 30분
  - 둘레길 산책
- 오전 11시
  - 체크아웃 이후 일정 조율 (저는 모각코 진행합니다)
  - 속초 온김에 뭐 하고 싶으신 분 의견 주세요
  - 최소 해떨어지기 전까지 서울 도착합니다.

### 비용

제가 부담하는 비용입니다.

- 워크샵 맥주+안주값
- 저녁 회값
- 뒷풀이 with 만석 닭강정

그 외 개인 부담입니다.
숙소 비용과 개인 비용 정도만 부담하시면 됩니다

**Comment by TaeHyoungKwon** [comment]
리조트 객실이 남은게 디럭스 더블룸 밖에 없네여 ㅠㅠ 저는 주변 펜션으로 잡겠습니다 ㅎㅎ

**Reply by TaeHyoungKwon** [reply]
속초 마린펜션으로 잡았습니다 ㅎㅎ 담날 차도 속초 고속버스 터미널에서 6시 45분 차가 있네여 ㅋㅋ

**Reply by TaeHyoungKwon** [reply]
제가 그날 당일이 마침 오전 9시부터 건강검진이더라구여, 오전 내에는 건강검진 끝내고 출발하면 얼추 3시 언저리에 숙소 도착할듯 합니다 저는 고속버스 타고 갈 예정입니다

추가로, 가볍게 즐길 수 있는 보드게임도 가져가겠습니다ㅋㅋㅋ

제 일정은

24일
오전 9시 -> 건강 검진
오후 3시 -> 숙소 도착
.
.
.
25일
오전 7시 -> 귀가(오후 1시 결혼식참가..)

입니다

**Reply by jongfeel** [reply]
따로 오시나요?
부담 되시면 제 차로 같이 가셔도 됩니다.

**Reply by TaeHyoungKwon** [reply]
아 네네 저는 고속버스 타고 따로 출발 할 예정입니다 ㅎㅎ

**Reply by Joseph-Cha** [reply]
저도 숙소는 일단 태형님과 같은 곳으로 잡았습니다 ㅎㅎ 
그리고 당일 연차 사용이 가능할 것 같고 채용 검진을 24일 오전에 받으려고 합니다.
오전 일찍부터 검진이 가능하다고 해서 12시까지 용산에 갈 수 있을 것 같아요(혹시 시간을 못맞출 것 같으면 말씀드리겠습니다!).

**Reply by jongfeel** [reply]
용산 오시면 제 차로 같이 가시죠! ㅋㅋ

**Comment by jongfeel** [comment]
1년이나 지났지만... 워크샵 사진을 올려봅니다.

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/970263f1-73a8-4459-9896-55e88dca012e)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/dd10a684-ee9c-4e19-be4f-d8517eeeafb3)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/63cb4b7c-87cd-4c84-9713-aed5844e75b1)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/57322bd9-e7c0-47ff-956e-3144bbd359fe)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/2afeea86-515c-466f-811b-9501a1cfc70f)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/cd0a458a-a1bb-4485-ae58-ae78303f7d39)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/847cab33-ec83-4cc2-8289-c49fe976bec3)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/1394e6ce-5aec-4620-b2ff-a048695679b6)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/7b92720b-a148-4402-af9b-23c283e1d31b)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/f98d9956-dfac-49c6-91f4-370cdf4be472)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/0e206aa4-89cb-4d22-b96c-b4f82755efac)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/e9766968-6a6a-481d-a39f-db59e0c30edc)

![image](https://github.com/ThinkAboutSoftware/AcademicConference/assets/17442457/6eede617-2f82-4034-8ec1-0aed800f9d82)
